src/parser/parser.rs

```diff
@@ -156,6 +200,7 @@ impl<'a> Parser<'a> {
         }
     }
 
+    /// Parses a built-in type, like [Type::void], and panics if the next token isn't one.
```

Comment:
```suggestion
    /// Parses a built-in type, like [Type::Void], and panics if the next token isn't one.
```

Pretty sure the type is capitalized